data.sql

```diff
@@ -3,4 +3,12 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 VALUES (1, 'Agumon', '2/3/2020', 0, true, 10.23),
 (2, 'Gabumon', '11/15/2018', 2, true, 8),
 (3, 'Pikachu', '1/7/2021', 1, false, 15.04),
-(4, 'Devimon', '5/12/2017', 5, true, 11);
+(4, 'Devimon', '5/12/2017', 5, true, 11),
+(5, 'Charmander', '2/8/2020', 0, false, 11),
+(6, 'Plantmon', '11/15/2021', 2, true, 5.7),
+(7, 'Squirtle', '4/2/1993', 3, false, 12.13),
+(8, 'Angemon', '6/12/2005', 1, true, 45),
+(9, 'Boarmon', '6/7/2005', 7, true, 20.4),
+(10, 'Blossom', '10/13/1998', 3, true, 17),
+(11, 'Ditto', '5/14/2022', 4, true, 22);
```

Comment:
- [ ] The weights of the following animals have negative values but you inserted them as positive values in this file:

- Charmander
- Plantmon
- Squirtle
- Angemon

#### Kindly change their values to negative ones. 👍 😎

Reply:
# Done
![Screenshot (193)](https://github.com/AleWaweru/vet-clinic-database/assets/102799430/7ef2f84b-266a-4dcf-b614-a24b78741a9a)

Reply:
solved